CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Support for symbolic link in file mode.
 - Make file type const `REGULAR_FILE_TYPE` `DIR_FILE_TYPE` `SYMBOLIC_LINK_FILE_TYPE` public, because `FileMode::file_type` is public, sometimes we need this const to determin file type.
+- Support for setting to a package a description different from the summary.
```

Comment:
The semantic signature of `PackageBuilder::new()` has changed, please mention that.  And typically I like to name the methods that have been added directly.

Reply:
Thanks, I've updated the changelog.